### ISSUE TITLE
DenseNet Settings

### ISSUE THREAD

**Issue by KingJamesSong** [deeplearning-wisc/gradnorm_ood#7]
Hi,

  Thanks for open-sourcing the codes.
  
  Could you please also reveal the code or the implementation details of the DenseNet-121 experiment? 

  DId you use the same resolution and the same temperature as the experiment of ResNetv2-121?

**Comment by iurgnauh** [comment]
Hi,

Thanks for being interested in our work!

For DenseNet-121, we used the PyTorch implementation: https://pytorch.org/vision/main/_modules/torchvision/models/densenet.html#densenet121.
And we used the pre-trained model provided by PyTorch: https://download.pytorch.org/models/densenet121-a639ec97.pth

The resolution we used here is a 224x224 center crop. Everything else (including temperature) stays the same.

Let me know if there is any issue in reproducing the results!
